src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.html

```diff
@@ -20,12 +20,14 @@ <h3 class="font-bolt order-title">{{ 'order-details.order-title' | translate }}<
             <span class="col-12 col-sm-2 form-group form-control">{{ bag.price }} {{ 'order-details.currency' | translate }}</span>
             <div class="col-12 col-sm-2 form-group count">
               <input
+                placeholder="0"
                 formControlName="quantity{{ bag.id }}"
                 type="number"
                 class="shadow-none form-control input"
                 min="0"
                 max="999"
-                (click)="calculate()"
+                [(ngModel)]="bag.quantity"
+                (change)="calculate()"
```

Comment:
calculate what ?
how about onQuantityChange() ?

Reply:
let also add check for 0 value 
so if the user type it will set the input value to null and placeholder will be visible

onQuantityChange(event): void {
    if(parseInt(event.target.value) === 0) {
      event.target.value = null
    }
    .....

Reply:
![image](https://user-images.githubusercontent.com/43093994/120101549-31fa5a80-c14f-11eb-9ad0-de64142cda60.png)